docs/static_site/src/.htaccess

```diff
@@ -3,6 +3,25 @@ DirectorySlash off
 RewriteEngine on
 RewriteOptions AllowNoSlash
 
+<IfModule mod_expires.c>
+  ExpiresActive on
+
+  # Images
+  ExpiresByType image/png                 "access plus 7 days"
+  ExpiresByType image/jpg                 "access plus 7 days"
+  ExpiresByType image/jpeg                "access plus 7 days"
+  ExpiresByType image/svg+xml             "access plus 7 days"
+
+  # CSS, Javascript, HTML
```

Comment:
Just curious, why are we caching JS & CSS but not HTML?

Reply:
Good question, it basically just preserves the current HTML "cache control" (no cache and verify diff by ETag) and extends CSS/JS cache control to 1 day. By not caching HTML, new features/contents can be published without delay.